File: src/main/java/io/github/svegon/capi/mixininterface/IMerchantInventory.java
package io.github.svegon.capi.mixininterface;

import net.minecraft.village.TradeOffer;

public interface IMerchantInventory {

    void setTradeOffer(TradeOffer tradeOffer);

    int getOfferIndex();

    void setMerchantRewardedExperience(int merchantRewardedExperience);
}

File: src/main/kotlin/io/github/svegon/mclientapi/mixininterface/IMerchantInventory.kt
package io.github.svegon.mclientapi.mixininterface

import net.minecraft.village.Merchant
import net.minecraft.village.TradeOffer

interface IMerchantInventory {
    val merchant: Merchant

    fun setTradeOffer(tradeOffer: TradeOffer?)

    val offerIndex: Int

    fun setMerchantRewardedExperience(merchantRewardedExperience: Int)
}
